GeneratedMonitoringArtifacts/Default/DefaultAlerts.json

```diff
@@ -45,10 +45,55 @@
+                            "description": "An extrapolation algorithm predicts that the file system will run out of space within the upcoming 24 hours. For more information on this alert, please refer to this [link](https://github.com/prometheus-operator/runbooks/blob/main/content/runbooks/node/NodeFilesystemSpaceFillingUp.md)."
+                        },
+                        "labels": {
+                          "severity": "warning"
+                        },
+                        "severity": 3,
+                        "resolveConfiguration": {
+                            "autoResolved": true,
+                            "timeToResolve": "PT10M"
+                        },
+                        "actions": [
+                            {
+                                "actionGroupId": "[parameters('actionGroupResourceId')]"
+                            }
+                        ]
+                    },
+                    {
+                        "alert": "NodeFilesystemSpaceUsageFull85Pct",
+                        "expression": "1 - avg by (namespace,cluster,job,device,instance,mountpoint)(node_filesystem_avail_bytes{job=\"node\"}) / avg by (namespace,cluster,job,device,instance,mountpoint)(node_filesystem_size_bytes{job=\"node\"}) > .85",
+                        "for": "PT15M",
+                        "annotations": {
+                            "description": "Disk space usage for node {{ $labels.instance }} on device {{ $labels.device }} is greater than 85%. For more information on this alert, please refer to this [link](https://github.com/prometheus-operator/runbooks/blob/main/content/runbooks/node/NodeFilesystemAlmostOutOfSpace.md)."
+                        },
+                        "labels": {
+                          "severity": "warning"
+                        },
+                        "severity": 3,
+                        "resolveConfiguration": {
+                            "autoResolved": true,
+                            "timeToResolve": "PT10M"
+                        },
+                        "actions": [
+                            {
+                                "actionGroupId": "[parameters('actionGroupResourceId')]"
+                            }
+                        ]
+                    },
                     {
                         "alert": "KubePodCrashLooping",
                         "expression": "max_over_time(kube_pod_container_status_waiting_reason{reason=\"CrashLoopBackOff\", job=\"kube-state-metrics\"}[5m]) >= 1",
                         "for": "PT15M",
+                        "annotations": {
+                            "description": "{{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is restarting {{ printf \"%.2f\" $value }} / second. For more information on this alert, please refer to this [link](https://github.com/prometheus-operator/runbooks/blob/main/content/runbooks/kubernetes/KubePodCrashLooping.md)."
```

Comment:
Thanks Soham, should we also add in the $cluster label in the descriptions too so that the customer knows which cluster this is happening on?

Reply:
Sounds good. I will add it. Thanks Grace!